project-memberships-tweak-auto-actions

From: Bryan Larsen <bryan@larsen.st>

First remove the actions we don't need on the `ProjectMembershipsController`:

SHOW_PATCH
---
 app/controllers/project_memberships_controller.rb |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/controllers/project_memberships_controller.rb b/app/controllers/project_memberships_controller.rb
index aa14a80..4e32ec5 100644
--- a/app/controllers/project_memberships_controller.rb
+++ b/app/controllers/project_memberships_controller.rb
@@ -2,6 +2,6 @@ class ProjectMembershipsController < ApplicationController
 
   hobo_model_controller
 
-  auto_actions :all
+  auto_actions :write_only
 
 end
